Vungle/VungleRouter.m

```diff
@@ -538,6 +538,28 @@ - (void)cleanupFullScreenDelegate:(id<VungleRouterDelegate>)delegate
     }
 }
 
+- (void)removeDelegatesIfContainsPlacement:(NSString *)placementID
+                                dictionary:(NSMutableDictionary *)dictionary
+{
+    NSArray *array = [dictionary.keyEnumerator allObjects];
+    for (NSString *key in array) {
+        if ([key containsString:placementID]) {
+            [dictionary removeObjectForKey:key];
+        }
+    }
+}
+
+- (void)removeDelegatesIfContainsPlacement:(NSString *)placementID
+                                     table:(NSMapTable *)table
+{
```

Comment:
Same as above, might we need to do thread synchronization here? Thanks a lot. :)

Reply:
Updated.